app.rb

```diff
@@ -184,6 +184,7 @@ def is_mongo_available?
   begin
     response = get_db_is_master
     return response.ok? && (response.documents.first['ismaster'] == true)
+      # \ || ENV["MONGOID_READ_MODE"] != 'primary'
```

Comment:
Initially we had this to report the database as operational when using a secondary in a replicaset, but it fails to create users, despite existing users being able to post, subscribe etc.